### PR TITLE
micronaut 2.0.0

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
failed on the audit check:

```
==> brew style homebrew/core
==> FAILED
== Formula/pandoc-include-code.rb ==
C: 13:  3: license (line 13) should be put before bottle (line 6)
```

Also there is an upstream issue reported on the build, https://github.com/micronaut-projects/micronaut-starter/issues/292

**Comment by chenrui333** [comment]
Rebased and refreshed the PR (after the `brew style` fix)

**Comment by SMillerDev** [comment]
Cool, it runs a bunch of unittests for no apparent reason.

**Comment by dawidd6** [comment]
```
22:26:34.284 [Test worker] ERROR o.t.d.DockerClientProviderStrategy - Could not find a valid Docker environment. Please check configuration. Attempted configurations were:
  22:26:34.291 [Test worker] ERROR o.t.d.DockerClientProviderStrategy -     UnixSocketClientProviderStrategy: failed with exception InvalidConfigurationException (ping failed). Root cause NoSuchFileException (/var/run/docker.sock)
  22:26:34.292 [Test worker] ERROR o.t.d.DockerClientProviderStrategy - As no valid configuration was found, execution cannot continue
```

**Comment by dawidd6** [comment]
https://github.com/Homebrew/homebrew-core/pull/56446/files

**Comment by chenrui333** [comment]
Nice context!

**Comment by alvarosanchez** [comment]
Can it be changed so that it doesn't run tests? Also, the right Gradle task to run would be `assemble`. Finally, note that it isn't necessary to depend on a Gradle formula, since the project contains a Gradle wrapper that will take care of using the appropriate Gradle version.

So the command would be:

    ./gradlew assemble -x test

**Comment by SMillerDev** [comment]
Thanks for the right command @alvarosanchez 

> Finally, note that it isn't necessary to depend on a Gradle formula, since the project contains a Gradle wrapper that will take care of using the appropriate Gradle version.

We try and avoid the gradle wrapper since it's important for us as a package manager to have everything work on the latest versions.

**Comment by SMillerDev** [comment]
It's now working locally jumping through some stupid hoops with unpacking zip files that the build creates. I guess that's what happens when you use GraalVM or something. I'd be happy if I never have to touch this mess ever again.

**Comment by alvarosanchez** [comment]
I don't know what are you referring to with "this mess"...

Also, the "stupid hoops with unpacking zip files" are unnecessary, since the `starter-cli/build/exploded` folder contains what you need.

Finally, I just realised there is a faster command to produce the artifacts: `gradle micronaut-cli:assemble -x test`

**Comment by SMillerDev** [comment]
> I don't know what are you referring to with "this mess"...

Working around GraalVM, because GraalVM is clearly not meant to be build from source.

> Also, the "stupid hoops with unpacking zip files" are unnecessary, since the starter-cli/build/exploded folder contains what you need.

Thanks for the pointer, it's really hard to navigate all the directories and build outputs without diving into gradle I guess.

> Finally, I just realised there is a faster command to produce the artifacts: gradle micronaut-cli:assemble -x test

I love faster, implementing it now

**Comment by alvarosanchez** [comment]
For the sake of clarity, GraalVM is not involved here. This formula builds a JAR from source, not a native binary.

File: Formula/micronaut.rb [hunk 1]
```diff
@@ -1,15 +1,22 @@
 class Micronaut < Formula
   desc "Modern JVM-based framework for building modular microservices"
   homepage "https://micronaut.io/"
-  url "https://github.com/micronaut-projects/micronaut-starter/archive/v2.0.0.RC1.tar.gz"
-  sha256 "91dc71b200c598a91b95a20792dcc3f3cb160b8e169b581f6b98689dc713acad"
+
```
Comment (chenrui333) [inline Formula/micronaut.rb:4]: Besides extra line, LGTM, sorry for missing this. :)
```suggestion

```

File: Formula/micronaut.rb [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+  url "https://github.com/micronaut-projects/micronaut-starter/archive/v2.0.0.tar.gz"
+  sha256 "5e30fc1878c9d7da9046745a5f71c611fc9bd1b765e32dd5e597a6fae1739f6c"
   license "Apache-2.0"
 
   depends_on "gradle" => :build
   depends_on "openjdk"
 
   def install
-    system "gradle", "build"
+    system "gradle", "micronaut-cli:assemble", "-x", "test"
```
Comment (chenrui333) [inline Formula/micronaut.rb:13]: This is really formula-specific step :D i wonder if they should make a new `gradle task` to abstract that away. :)

**Comment by graemerocher** [comment]
Any chance this will be merged soon? Users reporting issues

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@graemerocher should be being merged momentarily, thanks for checking. :)

**Comment by graemerocher** [comment]
Thanks!

**Pull request closed without merging**